package.json

```diff
@@ -38,18 +38,18 @@
     "prompt": "~1",
     "ramda": "~0",
     "sanitize-filename": "~1",
-    "slonik": "~23",
+    "slonik": "^23.6.0",
     "slonik-sql-tag-raw": "1.0.3",
     "tmp-promise": "~3",
     "uuid": "~3",
-    "yauzl": "~2.9"
+    "yauzl": "~2.10"
   },
   "devDependencies": {
     "app-root-path": "~2",
-    "eslint": "~8",
+    "eslint": "~7",
```

Comment:
This one looks like it's decreased :thinking:

Reply:
`eslint-config-airbnb-base` has dependency on eslint: "^5.16.0 || ^6.8.0 || ^7.2.0"

Reply:
It looks like the latest version of `eslint-config-airbnb-base` supports ESLint v8. What do you think about upgrading `eslint-config-airbnb-base` instead? That could also happen in a follow-up PR.

Reply:
`eslint-config-airbnb-base@15` has added new rules that impact lots of files in our codebase, I can handle that in a separate PR